src/components/ec-dropdown/ec-dropdown.spec.js

```diff
@@ -151,6 +151,17 @@ describe('EcDropdown', () => {
       });
       expect(wrapper.findByDataTest('ec-dropdown-search__cta-area').element).toMatchSnapshot();
     });
+
+    it('should forward CTA slot and have a tooltip on the CTA', () => {
```

Comment:
@jaimeperezpedra Could we say -> should forward CTA slot with tooltip?

Reply:
done!